### DISCUSSION TITLE
Raspberry mining info

### DISCUSSION THREAD

**Discussion by CryptoHyperProfit** [Chia-Network/chia-blockchain#9902] [General]
hello, I saw the Chia token project and I find it interesting. I would like to have information on mining. 
What is the best hardware configuration for mining with raspberry pi4? 
Do hard drives have to be SSD? 
What is the monthly mining performance with a raspberry pi 4?
Is there anyone who is mining well with raspberry and can help me in case of problems?
Which is the best miner for this token?
Thanks in advance
Regards

**Comment by Isanderthul** [comment]
>What is the best hardware configuration for mining with raspberry pi4?

An external drive attached via USB. 
> Do hard drives have to be SSD?

no. There are two steps making the plot file, and using the plot file. 
if all you have is a pi, and no computer, you can get an online service to make the plot files.
> What is the monthly mining performance with a raspberry pi 4?

It needs to be good enough to run your farmer https://youtu.be/gGla3Thya8w?t=606
> Which is the best miner for this token? 

Make the plots on your work pc, so its free.

**Reply by CryptoHyperProfit** [reply]
thanks what is online service to make the plot files.
